tardis/montecarlo/tests/test_cmontecarlo.py

```diff
@@ -119,8 +119,7 @@ def model():
         line_lists_j_blues=(c_double * 2)(*([1.e-10] * 2)),
         line_lists_j_blues_nd=0,
 
-        line_lists_Edotlu=(c_double * 3)(1e-10,1e-10,1.0), # Init to an explicit array 
-        line_lists_Edotlu_nd=0,
+        line_lists_Edotlu=(c_double * 3)(*[0.0,0.0,1.0]), # Init to an explicit array 
```

Comment:
To see that in properly increments in that case too, might as well test that right?